lib/kafka_ex/protocol/metadata.ex

```diff
@@ -1,5 +1,6 @@
 defmodule KafkaEx.Protocol.Metadata do
   alias KafkaEx.Protocol
+  import KafkaEx.Protocol.Common
```

Comment:
Random imports - meh. This is sort of a base class import, which I think is ok. However, an alias would work for me as well.